### PR TITLE
Update strategies.js for Preventing modification of the name and mail for CAS and Shibb

### PR DESCRIPTION
**Pull request by dooremont**
Preventing modification of the name and mail for CAS and Shibb

### CODE REVIEW

File: node_modules/oae-authentication/config/strategies.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ module.exports = {
         'description': 'Allow Shibboleth authentication for tenant',
         'elements': {
             'enabled': new Fields.Bool('Shibboleth Authentication Enabled', 'Allow Shibboleth authentication for tenant', false),
+            'disableProfile': new Fields.Bool('Preventing modification of the name and mail', 'Preventing modification of the name and mail', true),
```
Comment (nicolaasmatthijs) [inline node_modules/oae-authentication/config/strategies.js]: @dooremont : Is it necessary to disable modification of the email as well? What if CAS has returned an invalid email address?

**Comment by nicolaasmatthijs** [comment]
Left a question for @dooremont

**Comment by dooremont** [comment]
Normally in our operation that we check to IT then sends out the information into the administrators can prevent changes

**Comment by dooremont** [comment]
do you think added this modification for the next release ? 
Left a question for @nicolaasmatthijs

**Comment by dooremont** [comment]
replace by  #1313

**Pull request closed without merging**